windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# Certbot for Window (beta)
+
+The Certbot development team is proud to propose you the first beta release of Certbot for Windows.
+
+This document explains how to install Certbot and use it on Windows.
+
+## Supported features
+
+All usual operations to create and manage an account, manage existing certificates, or select the ACME server, are supported.
+
+In terms of authenticator plugins, `standalone`, `manual` and `webroot` are supported. DNS plugins will be available soon.
+
+For now, no installer plugins are available. `apache`, `nginx` will be available soon, and a new installer for IIS is under development.
+
+Certificate automated renewals are supported.
```

Comment:
I find

> Automated certificate renewals are supported.

slightly more idiomatic, although I don't think the original version is wrong